meta-lmp-base/recipes-bsp/u-boot/u-boot-fio-common.inc

```diff
@@ -12,12 +12,17 @@ UBOOT_REPO ??= "git://github.com/foundriesio/u-boot.git"
 
 SRC_URI = "${UBOOT_REPO};protocol=https;branch=${SRCBRANCH} \
     file://fw_env.config \
-    ${@bb.utils.contains('MACHINE_FEATURES', 'ebbr', 'file://lmp-ebbr.cfg', 'file://lmp-common.cfg', d)} \
+"
+
+# Handle lmp / lmp-mfgtool distro configs set
+SRC_URI:append = " \
+    ${@bb.utils.contains('MACHINE_FEATURES', 'ebbr', 'file://lmp-common-ebbr.cfg', 'file://lmp-common.cfg', d)} \
+    ${@bb.utils.contains('MACHINE_FEATURES', 'ebbr', 'file://lmp-ebbr.cfg', 'file://lmp.cfg', d)} \
     ${@bb.utils.contains('UBOOT_SIGN_ENABLE', '1', '', 'file://lmp-common-nosec.cfg', d)} \
-    ${@bb.utils.contains('MACHINE_FEATURES', 'ebbr', '', 'file://lmp.cfg', d)} \
 "
```

Comment:
I would rather have just one `ebbr`
```
# Handle lmp / lmp-mfgtool distro configs set
SRC_URI:append = " \
    ${@bb.utils.contains('MACHINE_FEATURES', 'ebbr',
        'file://lmp-common-ebbr.cfg', 'file://lmp-common.cfg', 'file://lmp-ebbr.cfg', 'file://lmp.cfg', d)} \
    ${@bb.utils.contains('UBOOT_SIGN_ENABLE', '1', '', 'file://lmp-common-nosec.cfg', d)} \
"
```

Reply:
What is the difference between `lmp-common-ebbr.cfg` and `lmp-ebbr.cfg`? Can we gather all ebbr configs in one cfg file only? (I'm asking because some times it's not possible)

Reply:
> What is the difference between `lmp-common-ebbr.cfg` and `lmp-ebbr.cfg`? Can we gather all ebbr configs in one cfg file only? (I'm asking because some times it's not possible)

@angolini 
lmp-common-ebbr.cfg - includes options common for all machines with enabled ebbr feature.
lmp-ebbr.cfg - an empty stub config file, like the same stub for lmp w/o ebbr feature `meta-lmp-base/recipes-bsp/u-boot/u-boot-fio/lmp.cfg`

We may need to rename lmp-common-ebbr.cfg to lmp-ebbr-common.cfg to move an accent. What do you think?

Reply:
> I would rather have just one `ebbr`
> 
> ```
> # Handle lmp / lmp-mfgtool distro configs set
> SRC_URI:append = " \
>     ${@bb.utils.contains('MACHINE_FEATURES', 'ebbr',
>         'file://lmp-common-ebbr.cfg', 'file://lmp-common.cfg', 'file://lmp-ebbr.cfg', 'file://lmp.cfg', d)} \
>     ${@bb.utils.contains('UBOOT_SIGN_ENABLE', '1', '', 'file://lmp-common-nosec.cfg', d)} \
> "
> ```

Could you please point me to a manual with such syntax?
As I see, the current code doesn't support such multiply result groups :(
https://github.com/openembedded/bitbake/blob/master/lib/bb/utils.py#L1022

Reply:
> @quaresmajose
> Could you please point me to a manual with such syntax?
> As I see, the current code doesn't support such multiply result groups :(
> https://github.com/openembedded/bitbake/blob/master/lib/bb/utils.py#L1022

@MrCry0 I understand it wrong and don't see the commas impact so please don't consider my suggestion.